src/platform/CMakeLists.txt

```diff
@@ -29,6 +27,9 @@ include_directories(platform
 target_link_libraries(platform
   update)
 
+target_link_libraries(platform
+  utils)
```

Comment:
Why not add to the above target_link_libraries?

Reply:
Not sure anymore to be honest, at first I added it to the first `target_link_libraries` but that created a conflict, so I separated it. But it could have gone there, let me do that.